eng/versioning/alternative_external_dependency_versions.json

```diff
@@ -0,0 +1,55 @@
+{
+  "jackson_2.10":
+  [
+    { "com.fasterxml.jackson.core:jackson-annotations": "2.10.0" },
+    { "com.fasterxml.jackson.core:jackson-annotations": "2.10.0" },
+    { "com.fasterxml.jackson.core:jackson-core": "2.10.0" },
+    { "com.fasterxml.jackson.core:jackson-databind": "2.10.0" },
+    { "com.fasterxml.jackson.dataformat:jackson-dataformat-avro": "2.10.0" },
+    { "com.fasterxml.jackson.dataformat:jackson-dataformat-xml": "2.10.0" },
+    { "com.fasterxml.jackson.datatype:jackson-datatype-jdk8": "2.10.0" },
+    { "com.fasterxml.jackson.datatype:jackson-datatype-jsr310": "2.10.0" },
+    { "com.fasterxml.jackson.module:jackson-module-parameter-names": "2.10.0" },
+    { "com.fasterxml.jackson.module:jackson-module-scala_2.12": "2.10.0" }
+  ],
+  "jackson_2.11":
+  [
+    { "com.fasterxml.jackson.core:jackson-annotations": "2.11.0" },
+    { "com.fasterxml.jackson.core:jackson-annotations": "2.11.0" },
+    { "com.fasterxml.jackson.core:jackson-core": "2.11.0" },
+    { "com.fasterxml.jackson.core:jackson-databind": "2.11.0" },
+    { "com.fasterxml.jackson.dataformat:jackson-dataformat-avro": "2.11.0"},
+    { "com.fasterxml.jackson.dataformat:jackson-dataformat-xml": "2.11.0" },
+    { "com.fasterxml.jackson.datatype:jackson-datatype-jdk8": "2.11.0" },
+    { "com.fasterxml.jackson.datatype:jackson-datatype-jsr310": "2.11.0" },
+    { "com.fasterxml.jackson.module:jackson-module-parameter-names": "2.11.0" },
+    { "com.fasterxml.jackson.module:jackson-module-scala_2.12": "2.11.0" }
+  ],
+  "jackson_2.12":
+  [
+    { "com.fasterxml.jackson.core:jackson-annotations": "2.12.0" },
+    { "com.fasterxml.jackson.core:jackson-annotations": "2.12.0" },
+    { "com.fasterxml.jackson.core:jackson-core": "2.12.0" },
+    { "com.fasterxml.jackson.core:jackson-databind": "2.12.0" },
+    { "com.fasterxml.jackson.dataformat:jackson-dataformat-avro": "2.12.0" },
+    { "com.fasterxml.jackson.dataformat:jackson-dataformat-xml": "2.12.0" },
+    { "com.fasterxml.jackson.datatype:jackson-datatype-jdk8": "2.12.0" },
+    { "com.fasterxml.jackson.datatype:jackson-datatype-jsr310": "2.12.0" },
+    { "com.fasterxml.jackson.module:jackson-module-parameter-names": "2.12.0" },
+    { "com.fasterxml.jackson.module:jackson-module-scala_2.12": "2.12.0" }
+  ],
+  "reactor_3.5":
```

Comment:
Let's use their BOM milestone names instead of Reactor given that Reactor and Reactor Netty aren't on the same version. So this would be Reactor 2022 and Reactor 3.4 would be Reactor 2020